lang/build.gradle

```diff
@@ -58,32 +58,34 @@ dependencies {
     testImplementation 'org.junit.jupiter:junit-jupiter-params:5.7.0'
 }
 
-sourceSets {
-    jmh {
-        java.srcDirs = ['src/jmh/java']
-        resources.srcDirs = ['src/jmh/resources']
-        compileClasspath += sourceSets.main.runtimeClasspath
-    }
-}
-
-def partiqlDomainFile = 'partiql-domains.kt'
```

Comment:
Could the .gitignore in the `lang/domains` folder also be deleted in this PR?

https://github.com/partiql/partiql-lang-kotlin/blob/5a073ca0a2ea4308aad7acd041d66fdc589dcef5/lang/src/org/partiql/lang/domains/.gitignore#L1-L2